[2025-04-13T17:33:43.850+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T17:33:43.862+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_forecast_ingestion_dag.clean scheduled__2025-04-13T16:00:00+00:00 [queued]>
[2025-04-13T17:33:43.867+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_forecast_ingestion_dag.clean scheduled__2025-04-13T16:00:00+00:00 [queued]>
[2025-04-13T17:33:43.867+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-13T17:33:43.872+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): clean> on 2025-04-13 16:00:00+00:00
[2025-04-13T17:33:43.875+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=535) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-13T17:33:43.876+0000] {standard_task_runner.py:72} INFO - Started process 536 to run task
[2025-04-13T17:33:43.876+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_forecast_ingestion_dag', 'clean', 'scheduled__2025-04-13T16:00:00+00:00', '--job-id', '928', '--raw', '--subdir', 'DAGS_FOLDER/weather_forecast_ingestion_dag.py', '--cfg-path', '/tmp/tmpof8rm166']
[2025-04-13T17:33:43.878+0000] {standard_task_runner.py:105} INFO - Job 928: Subtask clean
[2025-04-13T17:33:43.904+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_forecast_ingestion_dag.clean scheduled__2025-04-13T16:00:00+00:00 [running]> on host caedfbfc7792
[2025-04-13T17:33:43.950+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='beni' AIRFLOW_CTX_DAG_ID='weather_forecast_ingestion_dag' AIRFLOW_CTX_TASK_ID='clean' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T16:00:00+00:00'
[2025-04-13T17:33:43.951+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-13T17:33:45.163+0000] {logging_mixin.py:190} INFO - üóëÔ∏è Deleting outdated forecast: weather_forecast/bamberg/2025-04-13T1700.json
[2025-04-13T17:33:45.286+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-13T17:33:45.304+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-13T17:33:45.304+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_forecast_ingestion_dag, task_id=clean, run_id=scheduled__2025-04-13T16:00:00+00:00, execution_date=20250413T160000, start_date=20250413T173343, end_date=20250413T173345
[2025-04-13T17:33:45.342+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-13T17:33:45.356+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T17:33:45.357+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
